src/main/java/org/folio/service/impl/EcsTlrServiceImpl.java

```diff
@@ -90,6 +77,29 @@ public boolean delete(UUID requestId) {
     return false;
   }
 
+  private EcsTlr processCreationOfEcsTlr(EcsTlr ecsTlrDto) {
+    final EcsTlrEntity ecsTlr = requestsMapper.mapDtoToEntity(ecsTlrDto);
```

Comment:
**Minor**
Somewhere we use `final`, somewhere we don't. Let's remove everywhere as effectively almost all local variables are final.

Reply:
This block of code was moved as it was before my updates; however, I have removed it as you suggested